### PR TITLE
[Web PubSub] Update API for WPS Extensions library

### PR DESCRIPTION
**Pull request by Mohit-Chakraborty**

### CODE REVIEW

**Comment by Mohit-Chakraborty** [comment]
cc: @tg-msft

File: sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSub/samples/WebPubSubOutputBindingFunction.cs [hunk 1]
```diff
@@ -16,9 +16,9 @@ public static class WebPubSubOutputBindingFunction
         [FunctionName("WebPubSubOutputBindingFunction")]
         public static async Task RunAsync(
             [HttpTrigger(AuthorizationLevel.Anonymous, "get", "post")] HttpRequest req,
-            [WebPubSub(Hub = "hub", Connection = "<connection-string>")] IAsyncCollector<WebPubSubOperation> operation)
+            [WebPubSub(Hub = "hub", Connection = "<connection-string>")] IAsyncCollector<WebPubSubAction> operation)
```
Comment (tg-msft) [inline sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSub/samples/WebPubSubOutputBindingFunction.cs:19]: Nit - we should rename the parameter from `operation` to `action`

**Review by tg-msft — APPROVED** [review]
This looks great to me, but the `Operation` -> `Action` change needs approval from the WebPubSub team so I'm adding the "Do Not Merge" label to prevent anything from accidentally going in.

**Review by vicancy — APPROVED** [review]
Looks perfect to me!

**Pull request merged**